seas5/README.md

```diff
@@ -0,0 +1,26 @@
+## Usage: 
+
+The pipeline can be run locally from the command line by calling the following from the root level directory: 
+
+```
+python run_mars.py <scope> <start_year> <end_year>
```

Comment:
Good catch! I'm going to keep as just `seas5` for now, since we don't have any options besides mars. If we implement a CDS download then I think we could configure that as an argument, rather than through separate files.